javav2/example_code/sts/src/main/java/com/example/sts/AssumeRole.java

```diff
@@ -0,0 +1,118 @@
+// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+// snippet-sourcedescription:[AssumeRole.java demonstrates how to obtain a set of temporary security credentials.]
+// snippet-service:[AWS Security Token Service]
+// snippet-keyword:[Java]
+// snippet-keyword:[AWS Security Token Service]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[2020-09-21]
+// snippet-sourceauthor:[AWS-scmacdon]
+
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
```

Comment:
You can simplify this to : 
/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
SPDX-License-Identifier: Apache-2.0